spec/unit/installer/user_project_integrator/target_integrator_spec.rb

```diff
@@ -1,6 +1,13 @@
 require File.expand_path('../../../../spec_helper', __FILE__)
 
 module Pod
+  CHECK_MANIFEST_PHASE_NAME = Installer::UserProjectIntegrator::TargetIntegrator::BUILD_PHASE_PREFIX +
```

Comment:
please don't add constants in the tests -- it could cause us to accidentally depend on them in CocoaPods -- maybe an ivar in the `before`?